projects/packages/ip/composer.json

```diff
@@ -0,0 +1,55 @@
+{
+	"name": "automattic/jetpack-ip",
+	"description": "Utilities for working with IP addresses.",
+	"type": "jetpack-library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.4",
+		"automattic/jetpack-changelogger": "@dev",
+		"automattic/wordbless": "dev-master"
```

Comment:
Will we really need this? Or will `brain/monkey` be enough?

Reply:
Added WorDBless as a dep since the existing Brute Force Protection module tests use it, which we will want to port over. The IP functions reference a couple WordPress functions like `get_site_option` and `wp_unslash`.

That said, we could just mock those similar to `packages/waf` does, and remove the WorDBless dependency, keeping this lighter weight :+1:

Reply:
Removed via https://github.com/Automattic/jetpack/pull/28765/commits/63e42eee2d6e57a8f9cf2a776bbc2db0659b3b04